src/components/designs/prism/prism.css

```
@@ -0,0 +1,482 @@
+     field's hard luminance ceiling (see prism-field.ts + prism-css.test.ts). */
+  --pr-accent-text: #d4d0ff;
+  --pr-magenta: #e86bd0;
+  --pr-cyan: #66e0d6;
+  --pr-green: #9fe870;
+  --pr-error: #ffa08d;
+  /* Solid-enough dark fills for readable content over the moving field.
+     Plain rgba on purpose: backdrop-filter is banned (containment + cost). */
+  --pr-scrim: rgba(13, 10, 22, 0.78);
+  --pr-scrim-deep: rgba(13, 10, 22, 0.9);
+
+  --pr-serif: 'Instrument Serif', Georgia, 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)